src/pages/[guild]/collect/[chain]/[address].tsx

```diff
@@ -175,13 +182,37 @@ const Page = ({
                   <CollectibleImage src={image} isLoading={isLoading} />
 
                   <Stack spacing={6}>
-                    <Heading
-                      as="h2"
-                      fontFamily="display"
-                      fontSize={{ base: "3xl", lg: "4xl" }}
+                    <HStack
+                      flex="1 0 auto"
+                      justifyContent="flex-end"
+                      animation="slideFadeIn .2s"
```

Comment:
Looks cool, but since the most important part on the page is the NFT image, I wouldn't add an extra animation for the title

Reply:
I removed it